src/style-property-map-readonly.js

```diff
@@ -19,77 +19,20 @@
   }
 
   StylePropertyMapReadOnly.prototype.get = function(property) {
-    if (typeof property != 'string') {
-      throw new TypeError('parameter 1 is not of type \'string\'');
+    if (!propertyDictionary().isSupportedProperty(property)) {
+      throw new TypeError(property + ' is not a supported CSS property');
     }
 
-    var value = this._styleObject[property];
-    if (!value) {
+    propertyString = this._styleObject[property];
+    if (propertyString == '') {
```

Comment:
You can still write (and I think it's clearer since `'' == false == null` in javascript:

```javascript
if (!propertyString)
```